functions/go/apply-setter/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.15-alpine3.12
+ENV CGO_ENABLED=0
+WORKDIR /go/src/
+COPY . .
+RUN go build -v -o /usr/local/bin/function ./
```

Comment:
use a more descriptive name than function

Reply:
All the other functions are using same name.

https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/functions/go/set-annotation/Dockerfile
https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/functions/go/set-label/Dockerfile#L5